example/src/main/gateway/config/context-env.properties

```diff
@@ -0,0 +1,3 @@
+influxdb.influxdb=apim-service-metrics-runtime-influxdb.default
```

Comment:
I don't think this file should be here. It's env configuration.  And makeEnv.sh script in env repo will set these values instead